packages/workbox-background-sync/test/browser/request-queue.js

```diff
@@ -30,6 +30,14 @@ describe('request-queue tests', () => {
     callbacks,
   });
 
+  const getDelayedPromise = () => {
+    return new Promise((resolve)=>{
+      setTimeout(()=>{
```

Comment:
is this have to be taken care manually every time? just curious, is there a better way?